UndertaleModTool/Windows/FindReferencesTypesDialog/UndertaleResourceReferenceMethodsMap.cs

```diff
@@ -1577,6 +1590,9 @@ await Task.Run(() =>
                 funcReferences = null;
                 variReferences = null;
 
+                await mainWindow.StopProgressBarUpdater();
```

Comment:
Super small nitpick, but we could probably use a `finally` block for this instead of duplicating the code. Otherwise, this looks good to me.

Reply:
fixed